.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "inttypes.h": "c"
+    }
+}
```

Comment:
This looks like an addition related to choice of IDE/Editor.  Probably doesn't belong in this PR.

(I'm not adverse to adding some VSCode setup support, but that should be a separate PR I think).

Reply:
Oops. Added by mistake. Let me take that out.